_source/lib/collections/_helpers.coffee

```diff
@@ -1,34 +1,45 @@
-
 Apollos.schemas = {}
 
-Apollos.generateSchema = (name, schema) ->
+Apollos.generateSchema = (name, excludeTrackingFields, schema) ->
```

Comment:
I made sure it was not a breaking change, so you need not worry about that.  I have reordered the params to your preference.